### PR TITLE
Check if a file exists before writing to minimize data transfer

### PR DESCRIPTION
**Pull request by twsouthwick**

### CODE REVIEW

File: tools/PlateManager/Program.cs [hunk 1]
```diff
@@ -37,7 +38,7 @@ static Task<int> Main(string[] args)
             return root.InvokeAsync(args);
         }
 
-        static async Task Run(IEnumerable<FileInfo> file, Uri storage, bool interactive, string baseUrl)
+        static async Task Run(IEnumerable<FileInfo> file, Uri storage, bool interactive, string baseUrl, bool doNotSkipExisting)
```
Comment (coderandhiker) [inline tools/PlateManager/Program.cs:41]: I think "doNotSkipExisting" is less clear than "skipExisting", consider flipping the directionality of this variable

**Review by coderandhiker — COMMENTED** [review]
one variable naming nit

File: tools/PlateManager/Program.cs (hunk 1 above)
Comment (twsouthwick) [reply tools/PlateManager/Program.cs:41]: Fixed

**Review by coderandhiker — APPROVED** [review]
looks good to me

**Pull request merged**